packages/babel-core/src/config/full.js

```diff
@@ -12,7 +15,8 @@ import {
 } from "./config-chain";
 import type { UnloadedDescriptor } from "./config-descriptors";
 import traverse from "@babel/traverse";
-import { makeWeakCache, type CacheConfigurator } from "./caching";
+import { makeWeakCache, type CacheConfigurator } from "./caching-a";
+import { makeWeakCache as makeWeakCacheSyncSync } from "./caching";
```

Comment:
`makeWeakCacheSyncSync` => `makeWeakCacheSync`?

Reply:
I still have to refactor it, but there is a difference: `Sync` allows an async cache invalidation function and a sync cached function, while with `SyncSync` both are sinchronous.